src/content/hcp/tutorials-sidebar.json

```diff
@@ -45,6 +45,16 @@
 					"isBeta": true
```

Comment:
Good point in your questions, I think we should remove the `beta` tags from Boundary. Could totally be in a follow up PR though.

Reply:
there's also 'BETA' in a card section title on the[ tutorials landing page.  ](https://dev-portal-git-ambadd-hcp-waypoint-highlights-hashicorp.vercel.app/hcp/tutorials)
![Uploading Screen Shot 2022-10-07 at 3.02.45 PM.png…]()